gems/pending/VixDiskLib/VixDiskLib.rb

```diff
@@ -96,7 +96,10 @@ def self.setup_env
     vars_to_clear.each do |key|
       my_env.delete(key)
     end
-    my_env["LD_LIBRARY_PATH"] = my_env["LD_LIBRARY_PATH"] + ":" + VIXDISKLIB_PATH
+
+    my_env["LD_LIBRARY_PATH"] =  my_env["LD_LIBRARY_PATH"].nil? ?  VIXDISKLIB_PATH :
+                                (my_env["LD_LIBRARY_PATH"] + ":" + VIXDISKLIB_PATH)
```

Comment:
Or another way, is to split `my_env["LD_LIBRARY_PATH"]` by `:`, add `VIXDISKLIB_PATH` and then rejoin everything.

```ruby
my_env["LD_LIBRARY_PATH"] = (my_env["LD_LIBRARY_PATH"].to_s.split(':') << VIXDISKLIB_PATH).compact.join(":")
```